src/Components/Organisms/ContactFormModal/ContactFormModal.tsx

```diff
@@ -0,0 +1,134 @@
+import * as Yup from 'yup';
+import { Box } from '@mui/system';
+import {
+  Button,
+  LinearProgress,
+  Modal,
+  Stack,
+  Typography,
+} from '@mui/material';
+import { Field, Form, Formik } from 'formik';
+import { TextField } from 'formik-mui';
+
+import ClosingIcon from '../../Organisms/LoginForm/CloseIcon';
+
+const ContactFormValidation = Yup.object().shape({
+  email: Yup.string()
+    .required('Adres e-mail wymagany')
+    .email('Niepoprawny adres e-mail'),
+  message: Yup.string().required('Nie wysyłaj pustej wiadomości!'),
+});
+
+const ContactFormModal = (prop: { open: boolean; handleClose: () => void }) => {
+  return (
+    <Modal
+      open={prop.open}
+      onClose={prop.handleClose}
+      aria-describedby="modal-contact-form"
+    >
+      <Box
+        sx={{
+          position: 'absolute',
+          top: '50%',
+          left: '50%',
+          transform: 'translate(-50%, -50%)',
+          bgcolor: '#FFFFFF',
+          width: '36rem',
+          height: 'fit-content',
+          borderRadius: '1rem',
+          padding: ['2.5rem', '2.125rem'],
+          '@media screen and (max-width: 600px)': {
+            width: '19.625rem',
+            padding: ['2.5rem', '2.125rem'],
+          },
+        }}
+      >
+        <ClosingIcon handleClose={prop.handleClose} />
+        <Formik
+          initialValues={{
+            email: '',
+            message: '',
+          }}
+          validationSchema={ContactFormValidation}
+          onSubmit={(values, { setSubmitting }) => {
+            setTimeout(() => {
+              setSubmitting(false);
+              const data = JSON.stringify(values, null, 2);
+              console.log(data);
+              alert('Twoja wiadomość została wysłana');
```

Comment:
maybe will be good to add real functionality of sending mail? for example to sparalige@gmail.com

Reply:
I would give it a could label.